histomicstk/features/ComputeMorphometryFeatures.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import pandas as pd
+from skimage.measure import regionprops
+
+
+def ComputeMorphometryFeatures(im_label):
+    """
+    Calculates morphometry features for each object
+
+    Parameters
+    ----------
+    im_label : array_like
+        A labeled mask image wherein intensity of a pixel is the ID of the
+        object it belongs to. Non-zero values are considered to be foreground
+        objects.
+
+    rprops[i] : property
+        A rprops[i] property from regioprops.
+
+    Returns
+    -------
+    fdata: Pandas data frame containing the following morphometry features
+    for each object/label
+        'Area',
+        'Circularity',
+        'Eccentricity',
+        'EquivalentDiameter'
+        'Extent',
+        'MajorAxisLength',
+        'MinorAxisLength',
+        'MajorMinorAxisRatio',
+        'Perimeter',
+        'Solidity',
+    """
```

Comment:
I was going to suggest that we remove centroids. They certainly will be used (for calculating spatial features) but are not features themselves. Some segmentation algorithms also already provide their own seed points.